### DISCUSSION TITLE
vue3.0.0beta.9：a-upload组件，在上传头像时出现同时请求两次上传链接

### DISCUSSION THREAD

**Discussion by ismeljm** [vueComponent/ant-design-vue#5237] [General]
<a-upload
          name="account_avatar"
          list-type="picture-card"
          :show-upload-list="false"
          action="http://XXX"
        >
        <img v-if="userAvatar" :src="userAvatarUrl" alt="没找着您的头像，请重新上传" title="仅支持.jpg格式头像" />
        <div v-else>
            <loading-outlined v-if="uploading" />
            <plus-outlined v-else />
            <div class="ant-upload-text">上传头像</div>
        </div>
</a-upload>

**Comment by geeklibin** [comment]
两次请求中，如果有一次是options请求，那是正常的。

**Comment by MChungW** [comment]
请求可以改成手动上传的，这样会有一次请求
